extra_time_application/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+======================
+Extra Time Application
+======================
+This module adds functionality to the application time and
```

Comment:
This module adds functionality of time application to extend the remaining time in tasks and